modules/apps/commerce/commerce-payment-method-paypal/src/main/java/com/liferay/commerce/payment/method/paypal/internal/PayPalCommercePaymentIntegration.java

```diff
@@ -315,30 +319,26 @@ public CommercePaymentEntry refund(
 			PayPalHttpClient payPalHttpClient = _getPayPalHttpClient(
 				commercePaymentEntry);
 
-			CommerceCurrency commerceCurrency =
-				_commerceCurrencyLocalService.getCommerceCurrency(
-					commercePaymentEntry.getCompanyId(),
-					commercePaymentEntry.getCurrencyCode());
-
 			CapturesRefundRequest capturesRefundRequest =
 				new CapturesRefundRequest(
 					commercePaymentEntry.getTransactionCode()
 				).prefer(
 					"return=representation"
 				).requestBody(
-					new RefundRequest() {
-						{
-							amount(
-								new com.paypal.payments.Money(
-								).currencyCode(
-									commercePaymentEntry.getCurrencyCode()
-								).value(
-									_toScaledString(
-										commerceCurrency,
-										commercePaymentEntry.getAmount())
-								));
-						}
-					}
```

Comment:
Hey @brianchandotcom , I had to remove double brace initialization because they create hidden references that can't be serialized